### ISSUE TITLE
Basilisk V3 Pro Support pls?

### ISSUE THREAD

**Issue by nxbil29** [1kc/razer-macos#793]

**Comment by angelus2014** [comment]
Yes, that would be much appreciated as I've just received my Basilisk V3 Pro Wireless! I was a bit surprised that it's not supported as 'openrazer' states it among its supported devices......

**Comment by clemlesne** [comment]
Sounds related to https://github.com/1kc/razer-macos/issues/682.

**Comment by damirkotoric** [comment]
💯 pretty please add support for c3 pro

**Comment by damirkotoric** [comment]
💯 pretty please add support for v3 pro

**Comment by 0zxw0** [comment]
pretty please add support for v3 pro

**Comment by nikhilsharma81181** [comment]
Any update on Razor Basilisk V3 Pro support??

**Comment by clemlesne** [comment]
Should be closed for #682, we should discuss in the same canal?

**Comment by jakkornat** [comment]
Ready to review 🧑🏻‍💻: 
1. https://github.com/1kc/razer-macos/pull/818
2. https://github.com/1kc/librazermacos/pull/49

**Comment by nikhilsharma81181** [comment]
Use "OpenRGB" app for RGB customization and "SteerMouse 5" for button customization both app are available for mac,
I hope it helps for non-programmer/normal-users

**Comment by sergeyustyak** [comment]
I tried the version from the post below and my Basilisk V3 Pro determined by the app normally
https://github.com/1kc/razer-macos/issues/778#issuecomment-1579648255
